CoreFoundation/String.subproj/CFString.h

```diff
@@ -165,7 +165,11 @@ struct __CFConstStr {
         uint8_t _pad[4];
     } _base;
     uint8_t *_ptr;
+#if __LP64__
+    uint64_t _length;
+#else
```

Comment:
I have changed the code to use unsigned long instead (same as the length field in __CFString struct which is of type CFIndex)